app/javascript/components/admin/UserModal.js

```diff
@@ -33,16 +44,28 @@ class UserModal extends React.Component {
     this.setState({ roleTitle: data.value });
   };
 
-  handleLockedStatusChange = event => {
```

Comment:
is there a way to consolidate these `handleChange` functions to as few as possible?  I know this wasn't something you did but it seems we could utilize `let value = event.target.type === 'checkbox' ? event.target.checked : event.target.value;` to reduce the amount of handlers here

Reply:
if not - they're so simple by setting state you could do an inline function:
```
onChange={data => this.setState({ lockReason: data.value })}
```

Reply:
[7a3999aa66fe10463be631cec4e19d6d66b60e6d](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/7a3999aa66fe10463be631cec4e19d6d66b60e6d)